src/include/api/vamana_index.h

```diff
@@ -422,6 +436,7 @@ class IndexVamana {
       // @todo using index_type = size_t;
       auto dtype = vectors.feature_type();
 
+      // print distance function typr
```

Comment:
(Minor) Let's remove this, and also the places where we add newlines to different places in the code, please